packages/cli/src/init/element/element.ts

```diff
@@ -119,4 +119,51 @@ export function createElementGenerator(templateName: string):
           'Check out your new project README for information about what to do next.\n');
     }
   };
+
+  class Polymer3ElementGenerator extends ElementGenerator {
+    // TODO(bicknellr): Why doesn't this inherit properly? Because `async` is
+    // compiled out?
+    async prompting() {
+      return super.prompting();
+    }
```

Comment:
Can you double check that this doesn't work when this isn't here? I looked at the JS that's produced, and we're producing ES6 here. I can't see any reason why this code would do anything at all.

Reply:
It seems like async-await is being compiled out. I'm betting it has something to do with that.

Reply:
Also, yes, the parent class' `prompting` function isn't called if this isn't here. I can't figure out where yeoman-generator calls it though and I'm not sure how it gets called due to the async-await thing.

Reply:
Hmm, telling TypeScript to target ES2017 still has this problem.

Reply:
Here's the problem: https://github.com/yeoman/generator/blob/v1.0.1/lib/index.js#L382
Yeoman only checks the object's prototype's own properties, so I think I have to have something like this in here.

Reply:
I've added pass-through functions for all of the other methods as well.

Reply:
Oh hey, you found it too. I just filed https://github.com/yeoman/generator/issues/1065